### ISSUE TITLE
Gutenberg standalone editor does not work

### ISSUE THREAD

**Issue by saas-developer** [WordPress/gutenberg#23885]
**Describe the bug**

Gutenberg standalone editor does not work

Codesandbox: https://codesandbox.io/s/gutenberg-standalone-wvqf5

The code here is copied from `@wordpress/block-editor` npm package description -  https://www.npmjs.com/package/@wordpress/block-editor

**To reproduce**
1. Go to the sandbox.
2. Wait for modules and basic UI to load.
3. Click on `+` to add a block.
4. UI CRASH.

**Expected behavior**
Blocks list is displayed

** Screenshots
Initial Render
![InitialRender](https://user-images.githubusercontent.com/58209961/87255779-98e12900-c442-11ea-80bb-7b0f2414d571.png)

Issue

![Issue](https://user-images.githubusercontent.com/58209961/87255783-a0a0cd80-c442-11ea-9121-b05e10e4377f.png)

**Editor version (please complete the following information):**

List of packages from `package.json`

```
    "@wordpress/base-styles": "2.0.0",
    "@wordpress/block-editor": "4.3.0",
    "@wordpress/block-library": "2.22.0",
    "@wordpress/components": "10.0.0",
    "@wordpress/dom-ready": "2.10.0",
    "@wordpress/element": "2.16.0",
    "@wordpress/format-library": "1.22.0",
    "node-sass": "4.14.1",
```

**Desktop (please complete the following information):**
MAC OS.
Chrome latest, Firefox latest

**Comment by saas-developer** [comment]
This bug has been partially fixed by codesandbox - https://github.com/codesandbox/codesandbox-client/pull/4542 .

Now, the initial editor is rendered but when `+` is clicked to add a block, it throws error. Probably because of wrong usage. Maybe this could go to documentation?

Initial Render
![InitialRender](https://user-images.githubusercontent.com/58209961/87255779-98e12900-c442-11ea-80bb-7b0f2414d571.png)

Issue

![Issue](https://user-images.githubusercontent.com/58209961/87255783-a0a0cd80-c442-11ea-9121-b05e10e4377f.png)

**Comment by ktmn** [comment]
Can be temporarily fixed by adding settings to the editor provider:

```js
<BlockEditorProvider
  value={blocks}
  onInput={updateBlocks}
  onChange={updateBlocks}
  settings={{ __experimentalBlockPatterns: [] }}
>
```

**Comment by saas-developer** [comment]
Updated my codesandbox, that fixes the issue.
Thank You @ktmn

**Comment by talldan** [comment]
Hey @ktmn, Is there a separate issue tracking this? If not, it seems like this issue should be kept open.

**Comment by ktmn** [comment]
Yeah this issue should probably be kept open, the actual problem is somewhere else, [here](https://github.com/WordPress/gutenberg/blob/033536612f490c240eb0cbce36cbeaf456bb9111/packages/block-editor/src/components/inserter/quick-inserter.js#L136) or maybe [here](https://github.com/WordPress/gutenberg/blob/9b6fee52af1066294e4212c0affd87dfb197bc9b/packages/block-editor/src/components/inserter/hooks/use-patterns-state.js), no default value assigned to patterns.

**Comment by talldan** [comment]
This was fixed by #24554
